rocket/views.py

```diff
@@ -83,33 +84,28 @@ def post(self, request):
             user = form.save(commit=False)
             user.is_active = False
             user.save()
+            id=user.id
             try:
                 Userprofile.objects.create(
                     user=user, bio=form.cleaned_data['bio'], entryno=entrynum)
             except IntegrityError:
                 raise
                 # return HttpResponseRedirect('/signup')
             utils.send_confirm_email(user)
-            return HttpResponseRedirect('/confirmemail')
+            return HttpResponse("for activation go through http://127.0.0.1:8000/activate/?id=%s" %(id))
```

Comment:
Do you think this is the right way to do it? Also '127.0.0.1:8000' is localhost, what if I serve this app on some other url, will the message be still okay in that case?